content/source/docs/cloud/integrations/service-now/index.html.md

```diff
@@ -190,3 +194,84 @@ Access Control Roles | Description
 
 For users who only need to order from the Terraform Catalog, we recommend creating another role with read-only permissions for `x_terraform_vcs_repositories` to view the available repositories for ordering infrastructure.
 
+## Example Customizations
```

Comment:
This needs some reorganization, beyond just the prose style fixes. 

- It's extremely long, on an already overlong mono-page. This should be moved out into a new page, and the ServiceNow integration item in the sidebar should be turned into a section. While doing this, it's worth checking whether other things should move; note that this page bills itself as "setup instructions," so usage information probably belongs in a separate page. 
- The header levels are somewhat messed up -- we go directly from H3 to H5. Treat headers as indicators of the document's outline level, and only jump down one level at a time. 
- The navigation instructions are quite complex, and a couple of screenshots (1024px wide) would go a long way to help.